simulation-scripts/scenarios.yaml

```diff
@@ -5,91 +5,114 @@ scenarios:
     path: ./scenario/container-ambush
     id: container-ambush
     difficulty: Easy
+    description: "Your CI build agent is compromised; where can you get from here?"
   - name: Container Defeat in Detail
     path: ./scenario/container-defeat-in-detail
     id: container-defeat-in-detail
     difficulty: Medium
+    description: "A minor DaemonSet misconfiguration has led to cluster compromise; can you discover how?"
   - name: Container Phalanx Formation
     path: ./scenario/container-phalanx-formation
     id: container-phalanx-formation
     difficulty: Easy
+    description: "Use kubectl to investigate a logging bug that may be leaking secrets."
   - name: ETCD Inverted Wedge
     path: ./scenario/etcd-inverted-wedge
     id: etcd-inverted-wedge
     difficulty: Hard
+    description: "Secrets are leaking from your cluster; can you discover how?"
   - name: Master bait and bleed
     path: ./scenario/master-bait-and-bleed
     id: master-bait-and-bleed
     difficulty: Medium
+    description: "A pod can make calls it should not have permissions for; how serious is this?"
   - name: Master Encirclement
     path: ./scenario/master-encirclement
     id: master-encirclement
     difficulty: Medium
+    description: "A deployment should have been blocked by the cluster; how did it succeed?"
   - name: Network Feint
     path: ./scenario/network-feint
     id: network-feint
     difficulty: Medium
+    description: "Your service looks to be on the fritz, but is something more serious happening instead?"
   - name: Network Hammer and Anvil
     path: ./scenario/network-hammer-and-anvil
     id: network-hammer-and-anvil
     difficulty: Medium
+    description: "A developer has set up an odd SSH route; is it safe?"
   - name: Network Hedgehog Defense
     path: ./scenario/network-hedgehog-defence
     id: network-hedgehog-defence
     difficulty: Medium
+    description: "You are running a postmortem on a suspected attack; could they have crossed a namespace?"
   - name: Network Swarming
     path: ./scenario/network-swarming
     id: network-swarming
     difficulty: Easy
+    description: "Some odd audit logs have surfaced from your database; did someone manage to bypass your api?"
```

Comment:
The last word: API, should be in upper case I believe.

Reply:
I'd made the change, just hadn't pushed